core/common/modules/dev-tools/module.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Elementor\Core\Common\Modules\DevTools;
+
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+class Module extends BaseModule {
+	/**
+	 * @return \Elementor\Core\Base\Module|\Elementor\Core\Common\Devtools\Module
+	 */
+	public static function instance() {
+		return ( parent::instance() );
+	}
+
```

Comment:
?

Reply:
It's gives you the right instance with the right PHPDOC when using.

Reply:
Remove the brackets around the returned value and fix the DOC (it returns only `Elementor\Core\Common\Modules\DevTools\Module`).

Reply:
Remove it

Reply:
Handled.